apps/admin/package.json

```diff
@@ -9,8 +9,8 @@
     "dev": "dotenv -e ../../.env next dev -- -p 3001",
     "format": "prettier . --check --cache --cache-location=\"node_modules/.cache/prettiercache\"",
     "format:fix": "prettier . --write --cache --cache-location=\"node_modules/.cache/prettiercache\" --log-level=warn",
-    "lint": "eslint . --cache --cache-location \"node_modules/.cache/.eslintcache\" --max-warnings 0",
-    "lint:fix": "eslint . --fix --cache --cache-location \"node_modules/.cache/.eslintcache\" --max-warnings 0",
+    "lint": "eslint . --max-warnings 0",
+    "lint:fix": "eslint . --fix --max-warnings 0",
```

Comment:
[Disables ESLint caching because it doesn't work with typed rules](https://typescript-eslint.io/troubleshooting/faqs/eslint/#can-i-use-eslints---cache-with-typescript-eslint).

Reply:
does this impact anything perf wise?

Reply:
In theory, repeat runs of full project linting will be a little slower. Files that didn't change between runs now won't be re-linted.

In practice, because you're using typed linting, this shouldn't make much a difference. Most of the time in your lint runs is taken up by computing type information (i.e. running most of `tsc`). So I'd be surprised if this made a noticeable difference.

In other words, something like 95% of your time is in type info, and 5% in parsing. The caching shaves off maybe up to 4% of that parsing time. You've still got most or all of the type info.

Also, the caching is just ... blatantly broken by cross-file rules 😄. That includes `eslint-plugin-import`'s import analysis and typed linting. So even if the perf savings were meaningful, they're only attainable by skipping files that might need to be re-linted anyway!